### PR TITLE
Create header/section

### PR DESCRIPTION
**Pull request by aceDavon**
Created a personal portfolio template adding linters and necessary files for a basic HTML/CSS template.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./css/style.css" />
+  </head>
+  <body>
+    <header class="top-nav">
+      <div class="logo">
+        <img src="./img/icons/hamburger.png" alt="logo-icon" class="logo-icon" />
+        <span class="logo-txt"> Makoji David </span>
```
Comment (sumairq) [inline index.html]: - [ ] Logo should be an `<a>` tag, as it should also serve as a link to the homepage.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      </div>
+      <div class="menu">
+        <img src="./img/icons/email.png" alt="email-icon" class="mail-icon" />
+      </div>
+    </header>
```
Comment (sumairq) [inline index.html]: - [ ] The layout of the toolbar does not match the Figma design. Kindly change according to the Figma template

### Your Design
image

### Figma Design

image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./css/style.css" />
+  </head>
+  <body>
+    <header class="top-nav">
+      <div class="logo">
+        <img src="./img/icons/hamburger.png" alt="logo-icon" class="logo-icon" />
```
Comment (sumairq) [inline index.html]: - [ ] Kindly try to ensure that the hamburger icon is nested inside a `<nav>` tag.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,135 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: rgb(29, 44, 101);
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+}
```
Comment (sumairq) [inline css/style.css]: - [ ] Give this section a background image (decorations are always background images in CSS, not `` tags in HTML)

Comment (sumairq) [inline css/style.css]: - [ ] Let's make sure that the background image is properly showing. 

### Your Design

image


### Figma Design
image

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+  font-weight: 500;
+  color: #fff;
+  text-transform: capitalize;
+}
+
+.description {
+  color: #b3bac5;
+  margin-top: 45px;
+  font-weight: 400;
+}
```
Comment (sumairq) [inline css/style.css]: - [ ] Kindly fix the left and right margins and set them according to the Figma template.

### Your Design 

image


### Figma Design 

image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+        <img src="./img/icons/email.png" alt="email-icon" class="mail-icon" />
+      </div>
+    </header>
+    <main>
+      <section class="jumbotron">
+        <img src="./img/icons/circle-2-header.svg" alt="circle-icon" class="bottom-circle" />
+        <img src="./img/icons/circle-2-header.svg" alt="circle-icon" class="top-circle" />
+        <div class="section-txt">
+          <h1 class="title">Onate Makoji David</h1>
+          <span class="subtitle"> Software developer </span>
```
Comment (sumairq) [inline index.html]: - [ ] Let's make the subtitle an `<h2>` or `<h3>` tag.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+          <img src="./img/icons/Work.svg" alt="" />
+          <img id="arrow-down" src="./img/icons/down-arrow.png" alt="arrow-down-svg" />
+        </div>
+        <div class="social-links">
+          <img src="./img/icons/github.svg" alt="github-icon" />
+          <img src="./img/icons/linkedin.svg" alt="github-icon" />
+          <img src="./img/icons/angellist.svg" alt="github-icon" />
+          <img src="./img/icons/twitter.svg" alt="github-icon" />
+          <img src="./img/icons/medium.svg" alt="github-icon" />
+        </div>
```
Comment (sumairq) [inline index.html]: - [ ] Let's ensure that the list of icons uses a `<ul>` tag to group them, and `<li>` per icon; each icon has a link (`<a>` tag).

File: css/style.css [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+  color: #9390ff;
+  margin-top: 12px;
+}
+
+.lists {
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  gap: 6px;
+}
```
Comment (sumairq) [inline css/style.css:96]: - [ ] Let's make sure that the layout of the list of technologies matches the Figma template.  

### Your Design 

image


### Figma Template

image

**Review by sumairq — CHANGES_REQUESTED** [review]
### Hi @aceDavon , 


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 5 above)
Comment (aceDavon) [reply css/style.css]: @sumairq  Thanks for your time on the review. I have looked through the code again and resolved these issues you raised herein and would appreciate it if you could still go through the PR and review the changes. Thanks again for your feedback

Comment (sumairq) [reply css/style.css]: Hi, @aceDavon for re-review you need to re-submit this pull request in the dashboard. 😃

File: index.html [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./css/style.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="top-nav">
+        <a href="#" class="logo-txt"> Makoji David </a>
+        <div class="menu">
+          <img src="./img/icons/hamburger.png" alt="logo-icon" class="logo-icon" />
```
Comment (wuletawwonte) [inline index.html:15]: - [ ] [OPTIONAL] Please use the exact icon from the Figma design. For your information, you can export SVG icons from the Figma design.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+            <a href="#"><img src="./img/icons/linkedin.svg" alt="linkedIn-icon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./img/icons/angellist.svg" alt="angellist-icon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./img/icons/twitter.svg" alt="github-icon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./img/icons/medium.svg" alt="github-icon" /></a>
```
Comment (wuletawwonte) [inline index.html:72]: - [ ] [OPTIONAL] Please make sure the color of these social icons should resemble the Figma design. You can change the fill color of SVG files. 
- [ ] [OPTIONAL] Please make sure the layout of the list of technologies looks like the Figma design. 
- [ ] [OPTIONAL] Kindly add the `Work` button including the arrow icon before it. 

<p align="center">&nbsp;
 
| Your project |   Figma design |
| ---------- | ----------- |
 | socialicons | Template 4-main-mobile(1) |
</p>

**Review by wuletawwonte — APPROVED** [review]
### **STATUS: APPROVED  ✔️** 
------
## Hi @aceDavon 👋🏿👋🏿👋🏿,  

Your project is complete! Great job implementing the changes requested by the previous reviewer 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**